src/common/Navbar.jsx

```diff
@@ -1,81 +1,142 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { AppBar, Toolbar, Tabs, Tab, makeStyles } from '@material-ui/core';
+import { Hidden, AppBar, Toolbar, Box, Link, IconButton, Menu, MenuItem, makeStyles } from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
 
+import logo from '../assets/svg/logo.svg';
+import Image from './Image';
 
 const useStyles = makeStyles((theme) => ({
     root: {
-        background: '#79d4fd',
-        flexGrow: 1
+        background: '#79d4fd',  
     },
+    content: {
+        display: 'flex', 
+        justifyContent: 'space-between',
+    }, 
     logo: {
-        position: 'absolute', 
-        left: '10%',
-        padding: '0',
-        margin: '0'
-        
+        paddingLeft: theme.spacing(10),
     },
     navlinks: {
-        position: 'absolute',
-        right: '10%',
-        color: 'black',
-        
+        paddingRight: theme.spacing(10),
+        '& > *': {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
        "& > *": {
```